### DISCUSSION TITLE
Pyodide

### DISCUSSION THREAD

**Discussion by vfreysz** [Avaiga/taipy#102] [Announcements]
Can a taipy app use Pyodide?

**Comment by FlorianJacta** [comment]
@vfreysz Could you describe how you would want Taipy to interact with Pyodide?

**Comment by vfreysz** [comment]
It will be great if it could work like jupyterlite. So my taipy app will be running on the client side in the browser and not on the server.
For exemple i use it my lab to do some FFT on some data and i would like the computation to be done on the client side because my server is limited from a computational point of view and when i run this part of the code it slow down all the microservices.

**Reply by FlorianJacta** [reply]
It appears that Pyodide is unable to install certain dependencies required by Taipy, such as `gevent `(refer to https://github.com/pyodide/pyodide/issues/2664). Taipy relies on libraries that include C language components for improved performance. Regrettably, these packages are not currently supported in Pyodide. In order to use them, they need to be added to Pyodide's list of available packages.

**Comment by DeveloperVaibhav1** [comment]
> Can a taipy app use Pyodide?

Yes you can use Pyrodide in taipy
